source/includes/administration/_pricings.md

```diff
@@ -588,16 +651,28 @@ Attributes | &nbsp;
 `product.id`<br/>*UUID* | The id of the product to add prices.
 `cogs`<br/>*Map<string, double>* | Map from currency to price. Must include all currencies in pricing at that point (see Effective pricing API).
```

Comment:
@m-orfanos what about this? Why is map used here...

Reply:
I don't know who did that, did it doesn't make sense at all. The documentation doesn't even tell what values can be used.

Reply:
Also, "see Effective pricing API" should be changed to be an actual link to the API.

Reply:
If you do a search for `Map<`, only `pricings.md` file has them. These needs to be fixed.

Reply:
This should be renamed to "currency code". I don't know what "see Effective pricing API" refers to.

Reply:
Also `Map<string, double>` should be `Map`, the `<` and `>` don't make sense in this context, it's not Java code.

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/403/commits/bc146fc31b56590a678f6ae59e7320d19ec995b1